ground/src/main/java/com/google/android/ground/ui/home/mapcontainer/MapContainerFragment.java

```diff
@@ -219,21 +219,19 @@ private void moveToNewPosition(Point point) {
   }
 
   private void onBottomSheetStateChange(BottomSheetState state, MapFragment map) {
-    mapContainerViewModel.setSelectedLocationOfInterest(state.getLocationOfInterest());
+    Optional<LocationOfInterest> loi = Optional.ofNullable(state.getLocationOfInterest());
```

Comment:
Not relevant to this PR, but just a reminder: LOIs will soon not be able to be removed/modified from the Android client. There will be a separate flow for editing/deleting points captured as part of the data collection flow.

Reply:
I'll file a separate bug to clean it up. Thanks for pointing it out.